Mathlib/LinearAlgebra/Multilinear/Basic.lean

```diff
@@ -238,16 +229,17 @@ instance addCommMonoid : AddCommMonoid (MultilinearMap R M₁ M₂) :=
   coe_injective.addCommMonoid _ rfl (fun _ _ => rfl) fun _ _ => rfl
 #align multilinear_map.add_comm_monoid MultilinearMap.addCommMonoid
 
+/-- Coercion of a multilinear map to a function as an additive monoid homomorphism. -/
+def coeAddMonoidHom : MultilinearMap R M₁ M₂ →+ (((i : ι) → M₁ i) → M₂) :=
+  ⟨⟨FunLike.coe, rfl⟩, fun _ _ ↦ rfl⟩
```

Comment:
Can you use `where` syntax here to avoid the annoying nested `⟨`s that impede `extends` refactors?

Reply:
```suggestion
@[simps] def coeAddMonoidHom : MultilinearMap R M₁ M₂ →+ (((i : ι) → M₁ i) → M₂) where
  toFun := FunLike.coe; map_zero' := rfl; map_add' _ _ := rfl
```